examples/README.md

```diff
@@ -75,9 +66,11 @@ Sometimes you want to edit existing code examples that is live in the `/examples
 
 1. Edit code examples within the `/examples/<version_number>` directory.
 2. Commit and push the code examples to the repo.
-3. In Github Actions run a manual workflow with the `<version_number>` input of the Handsontable and Handsontable's wrapper version.
 4. The code examples will be deployed to the Github Pages and will be available under the same URL as they already were.
 
+To deploy the existing examples without commiting any changes:
+1. In Github Actions run a manual workflow with the `<version_number>` input of the Handsontable and Handsontable's wrapper version.
```

Comment:
This information actually repeats what's already in the section "Deployment of the new code examples".

Maybe you could just change that section to say that it is good for deploying from any branch, also when even when there were no new changes on the `develop` branch (because that's the case we are talking about here, if I understand correctly).

Also, please add the name of the workflow in the "Deployment of the new code examples" section and a link to it.

Reply:
1. I implied it can be done from any branch, but did not describe it as a desirable thing (as the `develop` should be generally used for that kind of thing), WDYT? <br> https://github.com/handsontable/handsontable/pull/9165/commits/5a486fa53cea65673c24677f8e069adb5002063a#diff-49aaa2819e35a856818ecec8c9fa7e1c79ad028d3f44bd749736353cfb51bac9R74
2. https://github.com/handsontable/handsontable/pull/9165/commits/5a486fa53cea65673c24677f8e069adb5002063a#diff-49aaa2819e35a856818ecec8c9fa7e1c79ad028d3f44bd749736353cfb51bac9R61